src/com/dabomstew/pkrandom/newgui/NewRandomizerGUI.java

```diff
@@ -2584,6 +2626,21 @@ private void romLoaded() {
             tpRegularTrainersCheckBox.setEnabled(false);
             tpRegularTrainersSpinner.setVisible(additionalPokemonAvailable);
 
+            boolean heldItemsSupport = pokemonGeneration == 4; // More gens coming after first review.
```

Comment:
I would prefer if this was called something like `trainerHeldItemsSupport` because we have multiple different types of held item randomization besides trainers.

Reply:
Done.